cfme/cloud/instance/__init__.py

```diff
@@ -114,6 +115,7 @@ def is_displayed(self):
     sidebar = View.nested(InstanceAccordion)
     search = View.nested(Search)
     including_entities = View.include(VMEntities, use_parent=True)
+    pagination = PaginationPane
```

Comment:
REQUIRED: 
1. if you add this pane, you have to initialize it.
2. it's called paginator everywhere. please rename it
3. I also just found same bug in virtual_machines. please fix it and rename pagination to paginator everywhere.

Reply:
Fixed.